docs/kubernetes/user-guides/snapshots.md

```diff
@@ -122,3 +122,141 @@
     lost+found
     sample-file.txt
     ```
+
+### Tips on Migrating from Alpha Snapshots
+
+The api version has changed between the alpha and beta releases of the CSI
+snapshotter. In many cases, snapshots created with the alpha will not be able to
+be used with a beta driver. In some cases, they can be migrated.
+
+*You may lose all data in your snapshot!*
+
+*These instructions not work in your case and could delete all data in your
+snapshot!*
+
+These instructions happened to work with a particular GKE configuration. Because
+of the variety of alpha deployments, it is not possible to verify these steps
+will work all of the time. The following has been tested with a GKE cluster of
+master version 1.17.5-gke.0 using the following image versions for the
+snapshot-controller and PD CSI driver:
+
+  * snapshot-controller:v2.0.1
+  * csi-provisioner:v1.5.0-gke.0
+  * gke.gcr.io/csi-attacher:v2.1.1-gke.0
+  * gke.gcr.io/csi-resizer:v0.4.0-gke.0
+  * gke.gcr.io/csi-snapshotter:v2.1.1-gke.0
+  * gke.gcr.io/gcp-compute-persistent-disk-csi-driver:v0.7.0-gke.0
+
+#### Overview
+
+The idea is to provision a `VolumeSnapshotContents` in your new or upgraded
+cluster with the PD snapshot handle from the alpha snapshot, then bind that to a
+new `VolumeSnapshot`.
+
+If you are able to set the `deletionPolicy` of any existing
+`VolumeSnapshotContents` to `Retain`, or have already thus set the
+d`deletionPolicy` of the creating `VolumeSnapshotClass`, then deleting the
```

Comment:
I don't remember if deletion of snapshots looks at the field from the VolumeSnapshotClass.

Maybe it's safer to just update all of them to Retain.

Reply:
Done. It worked for me with just changing the contents but you have a good point.